pyproject.toml

```diff
@@ -33,6 +29,5 @@ enable_error_code = [
 disallow_any_expr = true
 disallow_any_explicit = true
 
-
-[tool.isort]
```

Comment:
Should we align this with black? I think it makes sense to aim for common ground among all `checkmk` `pyproject.toml`s

Reply:
I would prefer to keep pyproject.toml as short as possible. Since this is an isolated project, I think it doesn't really matter if the formatting is consistent with other Checkmk projects.